sentinel5dl/__init__.py

```diff
@@ -181,4 +197,22 @@ def download(products, output_dir='.', logger_fn=None):
         if logger_fn:
             logger_fn(f'Downloading {uuid} to {filename}')
         path = f'/odata/v1/Products(\'{uuid}\')/$value'
+
+        # Get md5 sum
+        md5um_path = f'/odata/v1/Products(\'{uuid}\')/Checksum/Value/$value'
+        md5sum, _ = __http_request(md5um_path)
+        md5sum = md5sum.decode()
+
+        # Check if file exist
+        if os.path.exists(filename):
+            # Compare md5 sum
+            if __md5(filename) == md5sum:
+                if logger_fn:
+                    logger_fn(f'{filename} already exist. Skipping...')
```

Comment:
To be more consistent with the other log messages:

```suggestion
                    logger_fn(f'Skipping {filename} since it already exist.')
```